Fix new Flake8 errors

```
tests/CLI/modules/config_tests.py:70:13: H214: Use assertIn/NotIn(A, B) rather than assertTrue/False(A in/not in B) when checking collection contents.
tests/CLI/modules/config_tests.py:73:13: H214: Use assertIn/NotIn(A, B) rather than assertTrue/False(A in/not in B) when checking collection contents.
tests/CLI/modules/config_tests.py:74:13: H214: Use assertIn/NotIn(A, B) rather than assertTrue/False(A in/not in B) when checking collection contents.
tests/CLI/modules/config_tests.py:75:13: H214: Use assertIn/NotIn(A, B) rather than assertTrue/False(A in/not in B) when checking collection contents.
tests/CLI/modules/vs/vs_create_tests.py:669:13: H214: Use assertIn/NotIn(A, B) rather than assertTrue/False(A in/not in B) when checking collection contents.
```

those tests need to be updated

--- a/tests/CLI/modules/config_tests.py
+++ b/tests/CLI/modules/config_tests.py
@@ -67,13 +67,13 @@ class TestHelpSetup(testing.TestCase):
             result = self.run_command(['--config=%s' % config_file.name, 'config', 'setup'])
 
             self.assert_no_fail(result)
-            self.assertTrue('Configuration Updated Successfully' in result.output)
+            self.assertIn('Configuration Updated Successfully', result.output)
             contents = config_file.read().decode("utf-8")
 
-            self.assertTrue('[softlayer]' in contents)
-            self.assertTrue('username = user' in contents)
-            self.assertTrue('api_key = AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' in contents)
-            self.assertTrue('endpoint_url = %s' % consts.API_PUBLIC_ENDPOINT in contents)
+            self.assertIn('[softlayer]', contents)
+            self.assertIn('username = user', contents)
+            self.assertIn('api_key = AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA', contents)
+            self.assertIn('endpoint_url = %s' % consts.API_PUBLIC_ENDPOINT, contents)
 
     @mock.patch('SoftLayer.Client')
     @mock.patch('SoftLayer.CLI.formatting.confirm')

--- a/tests/CLI/modules/vs/vs_create_tests.py
+++ b/tests/CLI/modules/vs/vs_create_tests.py
@@ -666,8 +666,7 @@ class VirtCreateTests(testing.TestCase):
                                        '--domain', 'TESTING', '--flavor', 'B1_2X8X25',
                                        '--datacenter', 'TEST00', '--os', 'UBUNTU_LATEST'])
             self.assert_no_fail(result)
-            self.assertTrue('Successfully exported options to a template file.'
-                            in result.output)
+            self.assertIn('Successfully exported options to a template file.', result.output)
             contents = config_file.read().decode("utf-8")
             self.assertIn('hostname=TEST', contents)
             self.assertIn('flavor=B1_2X8X25', contents)

